components/navbar/SubNav.js

```diff
@@ -1,21 +1,18 @@
-import Router, { useRouter } from "next/router";
+import { useRouter } from "next/router";
 import Tabs from "@components/Tabs";
+import { usePathname } from "next/navigation";
```

Comment:
`next/navigation` is for use only with the `app` directory.
Since we use the `pages` directory, `router.pathname` should be used.

Reply:
oh good spot 👍